label_studio/data_import/api.py

```diff
@@ -327,7 +327,11 @@ class ImportPredictionsAPI(generics.CreateAPIView):
     def create(self, request, *args, **kwargs):
         # check project permissions
         project = self.get_object()
-        tasks_ids = set(Task.objects.filter(project=project).values_list('id', flat=True))
+
+        tasks_ids_and_project_ids = set(Task.objects.filter(project=project).values_list('id', 'project_id'))
```

Comment:
In the cases of projects which already have 10-20k tasks or more (some I've seen have 100k+ tasks), this will start to blow up the data import process and task creation process. 

I noticed we were already doing this, I guess I wonder if this isn't already a problem we face. Also to this point, won't the project_id be redundant and wasteful to make this grab the additional field which we already have?

```suggestion
        tasks_ids = set(Task.objects.filter(project=project).values_list('id', flat=True))
```

Reply:
Seriously, looking at this with a fresh set of eyes, I have no idea why I did it this way.  Looks like I'm filtering tasks on project and redundantly including projects on all of those tasks.  This can be greatly simplified.